libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/EmitEvent.cs

```diff
@@ -5,7 +5,7 @@
 using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Bot.Expressions.Properties;
+using AdaptiveExpressions.Properties;
```

Comment:
>AdaptiveExpressions [](start = 6, length = 19)

Likewise, Adaptive.Expressions

Reply:
Was reviewed in last adaptive cards working group meeting. AdaptiveExpressions was were we landed. @matthidinger.

Reply:
Haven't seen this, so we are landed on AdapativeExpresion, I added an approve on this